### ISSUE TITLE
Powerline does not track git branches

### ISSUE THREAD

**Issue by maciejadamski89** [powerline/powerline#2183]
https://askubuntu.com/questions/1373262/powerline-does-not-track-git-branches

**Comment by blackheaven** [comment]
I have the same issue with Python 3.8 / Powerline 2.8.2 on NixOS

**Comment by maciejadamski89** [comment]
Meanwhile I wil try re-create a problem on a docker image.

**Comment by maciejadamski89** [comment]
![2021-11-02 15_18_51-fish _mnt_c_Users_MaciejAdamski_Documents_Developer_about_me_page](https://user-images.githubusercontent.com/17511347/140657599-4b6ced22-65a4-4daa-aac3-19f787f38a5f.png)
It's look like this on mi side. @blackheaven please also share your screen shot.

**Comment by blackheaven** [comment]
![Screenshot_20211107_195248](https://user-images.githubusercontent.com/1362807/140658299-a5f434f4-d38b-4b7a-9633-c5d7b22138d3.png)

**Comment by maciejadamski89** [comment]
Yeah, look exactly as my problem. I also has `Python 3.8.10`. I tried disable all custom settings `.bashrc` `.config.fish` and it's not working nethier. Creating of new user same - doeas not fix the problem.

**Comment by blackheaven** [comment]
Actually, the deamon does track the change, disabling it fixes the segment in zsh, but vim needs to be reopened to have the branch updated.
